src/bse202/__init__.py

```diff
@@ -14,16 +14,25 @@
 
 def init_db():
     with app.app_context():
-        # start fresh
+        # Start fresh by deleting existing data
         delete_db()
```

Comment:
this is redundant, the sql files clear the DB, It won't cause issue (probably) but there's no point in adding this

Reply:
update made i think

Reply:
Could we remove the `delete_db` function?